modules/Blockchain/Polygon/index.js

```diff
@@ -0,0 +1,88 @@
+const Web3Implementation = require('../Web3Implementation');
+const path = require('path');
+const axios = require('axios');
+const constants = require('../../constants');
+
+
+const coinGeckoLink = 'https://api.coingecko.com/api/v3/simple/price?ids=origintrail&vs_currencies=usd';
+const gasStationLink = 'https://gasstation-mainnet.matic.network/';
+
+class Polygon extends Web3Implementation {
+    /**
+     * Initializing Polygon blockchain connector
+     */
+    constructor({ config, emitter, logger }, configuration) {
+        super({
+            config,
+            emitter,
+            logger,
+            contractPath: path.join(__dirname, 'abi'),
+        }, configuration);
+
+        this.logger.info(`[${this.getBlockchainId()}] Selected blockchain: Polygon`);
+    }
+
+    async getRelativeTracPrice() {
```

Comment:
This code is the same as it is in xDAI blockchain implementation. Can we move it to the separated service or in default implementation?

Reply:
Do you suggest to add `getRelativeTracPrice, getGasPrice, calculateGasPrice, and saveNewGasPriceAndTime` to the default web3 implementation?

Reply:
If it's possible let's move it to default web3 implementation